projects/commudle-admin/src/app/feature-modules/notifications/components/notifications-popover/notifications-popover.component.scss

```diff
@@ -0,0 +1,65 @@
+nb-card {
+  width: 500px;
+  max-height: 500px;
+  margin: 0;
+  border-radius: 0.25rem;
+  border: 0;
+
+  nb-card-body {
+    padding: 0;
+    nb-list {
+      nb-list-item {
+        position: relative;
+
+        &:hover {
+          background-color: honeydew;
```

Comment:
let's not use new colours, either we use colours we have already defined or from tailwind

Reply:
sure.